File: swift/Samples/people_picker_sample.swift
import FluentCards

func createPeoplePickerCard() -> Card {
    return AdaptiveCardBuilder()
        .withVersion("1.6")
        .addInputChoiceSet { i in
            i.withID("people-picker")
                .withLabel("Select users in the whole organization")
                .withIsMultiSelect(true)
                .withValue("user1,user2")
                .withChoicesData("graph.microsoft.com/users")
        }
        .addAction { a in
            a.submit("Submit")
        }
        .build()
}

File: rust/samples/src/people_picker_sample.rs
use fluent_cards::*;

pub fn create_people_picker_card() -> Card {
    AdaptiveCardBuilder::new()
        .with_version("1.6")
        .add_input_choice_set(|i| {
            i.with_id("people-picker")
                .with_label("Select users in the whole organization")
                .with_is_multi_select(true)
                .with_value("user1,user2")
                .with_choices_data("graph.microsoft.com/users");
        })
        .add_action(|a| {
            a.submit("Submit");
        })
        .build()
}
